docs/release_notes/december_2022.md

```diff
@@ -17,12 +17,14 @@ This page includes new features and improvements to the Airbyte Cloud and Airbyt
 ### Improvements
 * Improved the Connection UX by preventing users from modifying an existing connection if there is a breaking change in the source schema. Now users must review changes before modifying the connection. [#20276](https://github.com/airbytehq/airbyte/pull/20276)
 * Improved the stream catalog index by defining `stream`. This precaution keeps all streams matching correctly and data organized consistently. [#20443](https://github.com/airbytehq/airbyte/pull/20443)
-* Updated the `StandardDestinationDefintion` and moved normalization fields to a `normalizationConfig` object. [#20573](https://github.com/airbytehq/airbyte/pull/20573)
-* Updated the API to support column selection configuration. [#20259](https://github.com/airbytehq/airbyte/pull/20259)
+* Updated the API to support column selection configuration in Airbyte Cloud. [#20259](https://github.com/airbytehq/airbyte/pull/20259)
```

Comment:
@mfsiega-airbyte Here's what I have for the release note :) Let me know if you think we should make any changes.